runtime/libia2/include/ia2_internal.h

```diff
@@ -18,12 +18,17 @@ struct dl_phdr_info;
 #include <link.h>
 #include <locale.h>
 #include <stdbool.h>
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sys/mman.h>
 #include <unistd.h>
 
+#ifdef IA2_LIBC_COMPARTMENT
```

Comment:
All these `#ifdef IA2_LIBC_COMPARTMENT` should probably be `#if defined(IA2_LIBC_COMPARTMENT)` (it would make a difference when passing in `-DIA2_LIBC_COMPARTMENT=0`)

Reply:
I realize there's not much of a material difference on x86, but I think it more closely describes what you meant